mlflow/tracking/fluent.py

```diff
@@ -126,14 +118,6 @@ def start_run(run_id=None, experiment_id=None, source_name=None, source_version=
         user_specified_tags = {}
         if parent_run_id is not None:
```

Comment:
i remember vaguely we got rid of `parent_run_id` in other places. is setting it in `user_specified_tags` here the desired behavior?

Reply:
This is the auto-detection of parent_run_id. That property is no longer a parameter of start_run.